### PR TITLE
Makefile: Add retries for current instabilities of t/full-stack.t

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/98952

### CODE REVIEW

**Review by kalikiana — APPROVED** [review]
Fine with me. Let's just try to keep in mind that we also fix this, though.

**Review by kalikiana — CHANGES_REQUESTED** [review]
Actually, it doesn't seem like it's working:

    Cannot detect source of 'RETRY=5'! at /usr/lib/perl5/5.26.1/TAP/Parser/IteratorFactory.pm line 261.

File: Makefile [hunk 1]
```diff
@@ -176,7 +176,7 @@ test-unstable:
 
 .PHONY: test-fullstack
 test-fullstack:
-	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=9 PROVE_ARGS="$$HARNESS t/full-stack.t"
+	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=9 PROVE_ARGS="$$HARNESS RETRY=5 t/full-stack.t"
```
Comment (kalikiana) [inline Makefile:179]:
```suggestion
	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=9 RETRY=5 PROVE_ARGS="$$HARNESS t/full-stack.t"
```

Comment (okurz) [reply Makefile]: Duh, obviously. Thx. fixed

**Review by Martchus — APPROVED** [review]

**Pull request merged**